std/range/package.d

```diff
@@ -9213,3 +9221,292 @@ if (is(typeof(fun) == void) || isSomeFunction!fun)
 
     auto r = [1, 2, 3, 4].tee!func1.tee!func2;
 }
+
+/**
+Extend the length of the input range $(D r) by padding out the start of the
+range with the element $(D e). The element $(D e) must be of a common type with
+the element type of the range $(D r) as defined by $(REF CommonType, std, traits).
+If $(D n) is less than the length of of $(D r), then the contents of $(D r) are
+returned.
+
+If $(D r) is a string with Unicode characters in it, remember D's rules about the
```

Comment:
"remember" is too conversational, use "If ..., padLeft obeys D's rules about..."